synapse/models/orgs.py

```diff
@@ -1,9 +1,197 @@
 from synapse.common import guid
 
-import synapse.lib.tufo as s_tufo
 import synapse.lib.module as s_module
 
-class OuMod(s_module.CoreModule):
+
+class OuModule(s_module.CoreModule):
+    def getModelDefs(self):
+        modl = {
+            'types': (
+                ('ou:sic', ('str', {'regex': r'^[0-9]{4}$'}), {
+                    'doc': 'The four digit Standard Industrial Classification Code.',
+                    'ex': '0111',
+                }),
+                ('ou:naics', ('str', {'regex': r'^[1-9][0-9]{4}[0-9]?$'}), {
```

Comment:
Same comment as ou:sic

Reply:
correct - it actually made no sense as a integer since it is a prefix based specifier that just happens to use digitis.  ranging doesn't make sense on it.